src/clients/base-client.js

```diff
@@ -163,6 +184,7 @@ class BaseClient {
     const url = buildUrl(this.baseUrl, path, parameters)
 
     return this.sendRequest(url, {
+      body: parameters.data,
```

Comment:
I wish `delete` initially accepted `data` as a parameter, but... I guess introducing it now can break some apps (new parameter which is not the last in parameters)